deploy/basic/terraform/schema.yaml

```diff
@@ -70,6 +70,31 @@ groupings:
     - object_storage_mushop_media_compartment_ocid
     - object_storage_mushop_media_visibility
 
+  - title: "Advanced Resource Options - DNS"
```

Comment:
These field should only be visible if advanced options is selected

Reply:
This field is mutually exclusive with the "Use only Always Free Resources" option.

Reply:
I will update the schema so that all of the DNS and WAF features are not available if free resources is checked off.